spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe UsersController do
+describe Users::RegistrationsController do
```

Comment:
Since I wrote these tests, I figured I should make sure that they're compatible with this PR. When I ran them, the error message said that this block of tests needs to have this code inserted at (or near) the beginning:
```
  before do
    @request.env["devise.mapping"] = Devise.mappings[:user]
  end
```
In addition, all of the parameters use `user_registration` instead of `user` now.